packages/merge-styles/src/styleToClassName.ts

```diff
@@ -214,6 +214,10 @@ export interface IRegistration {
 }
 
 export function styleToRegistration(options: IStyleOptions, ...args: IStyle[]): IRegistration | undefined {
+  if (args.length > 1 && (args[1] as IStyleBaseArray)?.length === 0) {
```

Comment:
this check makes a lot assumption. it's cleaner to have the check in caller method like what you had originally

Reply:
moved back